Assets/Scripts/PlayerController.cs

```diff
@@ -21,6 +27,12 @@ void Update()
     {
         Run();
         FlipSprite();
+
+        // Ground check
+        if (jumpDetector.isHit && rb.velocity.y <= 0f)
+        {
+            jumpCount = 0;
+        }
```

Comment:
To keep consistency, let's make sure all code in the update method are functions :) in this case, we could create a method named `GroundCheck()` and call it on update.

Reply:
Yeah, it's more readable now :)